types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
It appears that this will result in a call to the `TransitionGroup` ctor, which is impossible to invoke when it is pulled in via an `import * as` import.